lightwood/config/config.py

```diff
@@ -12,17 +12,19 @@ class CONFIG:
 
     # Enable deterministic cuda flag and use seeds everywhere (static or based on features of the dataset)
     DETERMINISTIC = True
-    OVERSAMPLE = False
     SELFAWARE = True
-    HELPER_MIXERS = False
+    HELPER_MIXERS = True
     ENABLE_DROPOUT = True
 
     """Probabilistic FC layers"""
     USE_PROBABILISTIC_LINEAR = False # change weights in mixer to be probabilistic
 
-    """Bayesian Network"""
-    NUMBER_OF_PROBABILISTIC_MODELS = 2
-
     """Encoder options"""
     TRAIN_TO_PREDICT_TARGET = True
     MAX_ENCODER_TRAINING_TIME = 3600 * 2
+
+    # Flags bellow are deprecated but still here in case we want to use the feature again
+    OVERSAMPLE = False
+
+    """Bayesian Network"""
+    NUMBER_OF_PROBABILISTIC_MODELS = 2
```

Comment:
should we remove this in favor of USE_PROBABILISTIC_LINEAR

Reply:
Bayesian net is already removed, but I will remove the flag completely, sure.

I kept forgetting to look at improving accuracy with PLinear, I'll look into it once I'm done and/or give up on embedding layers + trainable encoders things (other PR, which is not ready)